maat-court-data-api/src/test/java/gov/uk/courtdata/dces/controller/ConcorContributionsRestControllerTest.java

```diff
@@ -38,23 +39,29 @@ class ConcorContributionsRestControllerTest {
     @Test
     void testContributionFileContent() throws Exception {
 
-        when(concorContributionsService.getConcorFiles(ConcorContributionStatus.ACTIVE))
-                .thenReturn(List.of("FirstXMLFile", "SecondXMLFile", "ThirdXMLFile"));
+        when(concorContributionsService.getConcorContributionFiles(ConcorContributionStatus.ACTIVE))
+                .thenReturn(List.of(
+                        ConcorContributionResponse.builder().concorContributionId(1).xmlContent("FirstXMLFile").build(),
+                        ConcorContributionResponse.builder().concorContributionId(2).xmlContent("SecondXMLFile").build(),
+                        ConcorContributionResponse.builder().concorContributionId(3).xmlContent("ThirdXMLFile").build()));
 
         mvc.perform(MockMvcRequestBuilders.get(String.format(ENDPOINT_URL  +"/concor-contribution-files"))
                         .queryParam("status", ConcorContributionStatus.ACTIVE.name())
                         .contentType(MediaType.APPLICATION_JSON_VALUE))
                 .andExpect(status().isOk())
                 .andExpect(jsonPath("$", hasSize(3)))
-                .andExpect(jsonPath("$[0]").value("FirstXMLFile"))
-                .andExpect(jsonPath("$[1]").value("SecondXMLFile"))
-                .andExpect(jsonPath("$[2]").value("ThirdXMLFile"));
+                .andExpect(jsonPath("$[0].concorContributionId").value("1"))
```

Comment:
These assertions are very thorough (which is good!) although making an assumption on the order of the elements in the JSON response make this test a bit brittle and likely to break even if there is no functional change to the endpoint.

You could instead use `JSONAssert`
i.e.
`JSONAssert.assertEquals(expected, actual, JSONCompareMode.NON_EXTENSIBLE);
`

Or change the criteria of the JSON Paths to select elements based on concorContributionId
i.e.
`$[?(@.concorContributionId == 1)]
`

(My preference would be for JSONAssert as it validates the whole of the JSON response and is IMHO easier to read, but the choice is yours)

e.g. This test will fail for the below JSON even though it is functionally equivalent.
```
[
  {
    "concorContributionId": 3,
    "xmlContent": "ThirdXMLFile"
  },
  {
    "concorContributionId": 1,
    "xmlContent": "FirstXMLFile"
  },
  {
    "concorContributionId": 2,
    "xmlContent": "SecondXMLFile"
  }
]
```